docs/content/specs/shared/_index.md

```diff
@@ -667,8 +667,32 @@ For most scenario this is the command you'll need to call the below PowerShell s
 Set-AvmGitHubLabels.ps1 -RepositoryName "Org/MyGitHubRepo" -CreateCsvLabelExports $false -NoUserPrompts $true
 ```
 
+```shell
+# Linux / MacOs
```

Comment:
Is this not for a docker run and not specifically Linux/macos? As powershell can be installed cross platform. 

If so do we also need to provide guidance to install docker or a link?

Reply:
Correct I put this to mention explicitly that the code only work for Linux / MacOs. Less an issue here as it is running within a container.